grub-mkconfig.md

```diff
@@ -0,0 +1,8 @@
+# grub-mkconfig
+
+> Generate a GRUB configuration file.
+> More information: <https://www.gnu.org/software/grub/manual/grub/html_node/Invoking-grub_002dmkconfig.html>
+
+- Generate the configuration:
```

Comment:
```suggestion
- Generate a configuration file:
```

Reply:
@Managor I updated this suggestion to comply with our previous discussion.  You can apply it.